leasing/tests/report/test_reports.py

```diff
@@ -228,3 +228,174 @@ def test_laske_invoice_count_report(
         "2024-07-01": {"invoice_count": 0, "estimate_count": 1},
         "2024-08-01": {"invoice_count": 0, "estimate_count": 1},
     }
+
+
+def test_set_active_rent_period():
+    from leasing.report.utils import set_active_rent_period
```

Comment:
I guess this import belongs on the top of the file, or is there some reason for it to be here?